packages/frontend/src/creation-form/constraint-values-form/single-value.tsx

```diff
@@ -29,18 +29,19 @@ export const SingleValueConstraintForm = ({
     useEffect(() => {
         let valueErrorText = "";
 
-        if (!value0) {
+        if (value0 === undefined) {
             valueErrorText = t("error.value0.single.missing");
         } else if (
-            type?.value === ConstraintType.LOWER_THAN &&
-            BigInt(valueErrorText) <= 0n
+            type &&
+            type.value === ConstraintType.LOWER_THAN &&
+            value0 <= 0n
```

Comment:
This shouldn't really be possible right? There's the `allowNegative` prop that should be set to `false` on all `NumberInput`s and it shouldn't allow users to enter negative numbers. Still, this additional check is probably good to have.

Reply:
Ye we can leave the check I guess.